### ISSUE TITLE
Graphics corruption in Battles (Set 2).mra

### ISSUE THREAD

**Issue by birdybro** [MiSTer-devel/Arcade-Xevious_MiSTer#22]
![20210918_074807-screen_Interpolation (Sharp)](https://user-images.githubusercontent.com/16388068/133891210-5cad7d76-ca45-4d09-b320-a5049d21ce13.png)

I'll look into the MRA now, but posting here in case I can't figure it out.

**Comment by sorgelig** [comment]
Other Sets may require specific hardware.
If there is other set is working, then it should be enough.

**Comment by birdybro** [comment]
Yup, the other set works. It's not a major issue, just something that could be fixed later on if anyone feels like it. Nothing looked wrong in the MRA, so  this is just some functionality specific to Set 2 that isn't reproduced in the core.
